support-workers/src/test/scala/com/gu/zuora/Fixtures.scala

```diff
@@ -54,6 +57,21 @@ object Fixtures {
     Subscription(date, date, date)
   )
 
+  val touchpointEnvironment = TouchPointEnvironments.fromStage(Configuration.stage)
+  def getRatePlans[T <: Product](product: T): Seq[ProductRatePlan[Product]] = product.ratePlans.getOrElse(touchpointEnvironment, Nil)
+  val everydayHDProductRatePlanId = getRatePlans(catalog.Paper).find(x => x.fulfilmentOptions == HomeDelivery && x.productOptions == Everyday) map (_.id)
```

Comment:
I think you could replace this with `catalog.Paper.getProductRatePlan(touchpointEnvironment, Monthly, HomeDelivery, Everyday)`

Reply:
Good idea. Thanks.